core/domain/exp_domain.py

```diff
@@ -2609,3 +2610,4 @@ def __init__(self, exploration_id, title, category, objective,
         self.version = version
         self.exploration_model_created_on = exploration_model_created_on
         self.exploration_model_last_updated = exploration_model_last_updated
+        self.exp_model_first_published_msec = exp_model_first_published_msec
```

Comment:
Aha -- I now understand your earlier question about exploration_model_first_published_msec. Actually, this should just be called self.first_published_msec.

Here's why there are two fields with an exploration_model prefix: If you look at the base storage model, you'll see that a created_on and last_updated property are automatically set for every datastore model. So we use a different name here in order not to mix up creation/last-updated times for the summary model with the corresponding ones for the exploration model.